tests/by-util/test_uptime.rs

```diff
@@ -22,6 +36,221 @@ fn test_uptime() {
     // Don't check for users as it doesn't show in some CI
 }
 
+/// Checks for files without utmpx records for which boot time cannot be calculated
+#[test]
+#[cfg(not(any(target_os = "openbsd", target_os = "freebsd")))]
+// Disabled for freebsd, since it doesn't use the utmpxname() sys call to change the default utmpx
+// file that is accessed using getutxent()
+fn test_uptime_for_file_without_utmpx_records() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.write("file1", "hello");
+
+    ucmd.arg(at.plus_as_string("file1"))
+        .fails()
+        .stderr_contains("uptime: couldn't get boot time")
+        .stdout_contains("up ???? days ??:??")
+        .stdout_contains("load average");
+}
+
+/// Checks whether uptime displays the correct stderr msg when its called with a fifo
+#[test]
+#[cfg(target_os = "linux")]
+#[ignore = "disabled until fixed"]
```

Comment:
please create a bug to detail what needs to be fixed 
and add the bug number here :)

Reply:
Well, i changed the way the test works to fix it instead.